src/features/common/LandingVideo/index.tsx

```diff
@@ -27,34 +27,34 @@ function VideoContainer({ setshowVideo }: Props): ReactElement {
       switch (localStorage.getItem('language')) {
         case 'de':
           setvideoURL(
-            `${process.env.CDN_URL}/landing-vid/planet/en-intro-mobile-planet.mp4`
+            `https://a.plant-for-the-planet.org/wp-content/videos/en-intro-mobile-planet.mp4`
```

Comment:
How does this fit with https://github.com/Plant-for-the-Planet-org/plant-for-the-planet-org.github.io/pull/22/files?

Reply:
@norbertschuler file size limits, still testing.

Reply:
Problem is mainly Safari iOS, video does not load automatically.